core/src/test/java/apoc/schema/SchemasTest.java

```diff
@@ -392,12 +398,12 @@ public void testRelationshipExists() {
         db.executeTransactionally("CREATE INDEX rel_index_composite FOR ()-[r:PURCHASED]-() ON (r.date, r.amount)");
         awaitIndexesOnline();
 
-        assertTrue(singleResultFirstColumn(db, "RETURN apoc.schema.relationship.indexExists('KNOWS', ['since'])"));
```

Comment:
Why is this assertion removed? It feels like is should still be true, doesn't it?

Reply:
Fixed: 3cb7e8f